### PR TITLE
Modify the submission form to change the corresponding interface to allow for inputting multiple repositoryURL

### PR DESCRIPTION
**Pull request by nathanbaleeta**
- Update schema to support multiple software repositories

### CODE REVIEW

File: schemas/schema.js [hunk 1]
```diff
@@ -127,21 +127,44 @@ const schema = {
...
+                  isRequired: true,
+                  validate: [
+                    {
+                      type: "required",
+                    },
+                  ],
                 },
                 {
-                  type: "url",
+                  name: "repoURL",
```
Comment (lacabra) [inline schemas/schema.js:154]: As per your own changes to the candidates schema, this should be `url`, refer to [this line](https://github.com/DPGAlliance/publicgoods-candidates/blob/8ec299423ecc43c0bbff78967349ec191fac0ec6/nominee-schema.json#L349).

**Review by lacabra — CHANGES_REQUESTED** [review]

**Comment by lacabra** [comment]
@nathanbaleeta Now that DPGAlliance/publicgoods-candidates#922 has been merged, you should be able to fix the failing CI.

File: schemas/schema.js (hunk 1 above)
Comment (nathanbaleeta) [reply schemas/schema.js]: Fixed in commit f88f06b

**Review by lacabra — APPROVED** [review]
This looks good to me now, thanks @nathanbaleeta 🙏

**Pull request merged**